composer.json

```diff
@@ -48,5 +50,12 @@
         "cs": "phpcs",
         "cs-fix": "phpcbf",
         "test": "phpunit"
+    },
+    "extra": {
+        "zf": {
```

Comment:
We still need it, but `Zend\ComponentInstaller` will automatically register our `ConfigProvider` there.